iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractTckSuiteExhaustivityTest.java

```diff
@@ -0,0 +1,84 @@
+package com.powsybl.iidm.network.tck;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.params.ParameterizedTest;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public abstract class AbstractTckSuiteExhaustivityTest {
+
+    @Test
+    void assertAllTckTestsAreInstanced() {
+        ClassLoader classLoader = ClassLoader.getSystemClassLoader();
+        String tckPackageName = "com.powsybl.iidm.network.tck";
+        List<String> tckTestClasses = getPackageClasses(classLoader, tckPackageName)
+                .filter(clazz -> containsTest(classLoader, clazz))
+                .filter(clazz -> !clazz.contains("$"))
```

Comment:
It would be more performant to invert the conditions (since `containsTest` uses reflexivity) and to use a single `filter` clause instead of 2 (see this [page](https://www.baeldung.com/java-streams-multiple-filters-vs-condition#performance)).